pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -1418,3 +1435,248 @@ func (vs *VSphere) GetZoneByNodeName(ctx context.Context, nodeName k8stypes.Node
 func (vs *VSphere) GetZoneByProviderID(ctx context.Context, providerID string) (cloudprovider.Zone, error) {
 	return cloudprovider.Zone{}, cloudprovider.NotImplemented
 }
+
+// GetVolumeLabels returns the well known zone and region labels for given volume
+func (vs *VSphere) GetVolumeLabels(volumePath string) (map[string]string, error) {
```

Comment:
Should use `GetLabelsForVolume` as per https://github.com/kubernetes/cloud-provider/blob/master/cloud.go#L232-L235. Other providers will do the same once https://github.com/kubernetes/kubernetes/pull/73102 is merged

Reply:
Done.